cmake/modules/FindTensorFlow.cmake

```diff
@@ -4,8 +4,13 @@
 include(FindPackageHandleStandardArgs)
 
 find_path(TF_INCLUDE_DIR
-  NAMES tensorflow/c
+  NAMES third_party/tensorflow/c tensorflow/c
   HINTS ${SWIFT_TENSORFLOW_TARGET_INCLUDE_DIR} /usr/include /usr/local/include)
+if (EXISTS ${TF_INCLUDE_DIR}/third_party/tensorflow/c/c_api.h)
+  set(TF_PATH_ADJUSTMENT "third_party")
+else()
+  set(TF_PATH_ADJUSTMENT "")
```

Comment:
consider adding a comment that the else case is the normal setup, for "mainstream" development, CI, etc.

The "if" case is experimental, and not covered by CI as intended.